KachingPlugIn/Factories/L10nStringFactory.cs

```diff
@@ -34,12 +35,23 @@ public L10nString GetLocalizedString(CatalogContentBase content, string property
                 var localizedContent = _contentLoader.Get<CatalogContentBase>(content.ContentLink, culture);
 
                 PropertyData data = localizedContent.Property[propertyName];
-                if (data == null || data.IsNull || !(data.Value is string))
+                if (data == null || data.IsNull)
                 {
                     continue;
                 }
 
-                languageDictionary[culture.TwoLetterISOLanguageName] = (string)data.Value;
+                if (data.Value is string)
+                {
+                    languageDictionary[culture.TwoLetterISOLanguageName] = (string)data.Value;
+                }
+                else if (data.Value is XhtmlString)
+                {
+                    XhtmlString value = data.Value as XhtmlString;
```

Comment:
You can do (data.Value is XhtmlString xhtmlString), and skip line 49. Same goes for line 43.

Reply:
Neat, thanks